liblangutil/Token.h

```diff
@@ -270,6 +270,7 @@ namespace solidity::langutil
 	\
 	/* Yul-specific tokens, but not keywords. */                       \
 	T(Leave, "leave", 0)                                               \
+	T(RightArrow, "->", 0)                                             \
```

Comment:
We already have an `Arrow` token for `=>`. Should think about an alternative/better name, and no, not `YulArrow`.